{{ cookiecutter.repo_name }}/setup.py

```diff
@@ -15,10 +15,9 @@ def read(*paths):
         return f.read()
 
 requires = [
-    'Django==1.7.6',
+    'Django>1.8',
```

Comment:
Please use `Django==1.8.2`. Usually projects will not upgrade dependencies automatically.